release/src/main/python-release/python_release_automation.sh

```diff
@@ -19,7 +19,7 @@
 source release/src/main/python-release/run_release_candidate_python_quickstart.sh
 source release/src/main/python-release/run_release_candidate_python_mobile_gaming.sh
 
-for version in 2.7 3.5 3.6 3.7
+for version in 2.7 3.5 3.6 3.7 3.8
```

Comment:
Should we also add python-3.8 in release/src/main/scripts/build_release_candidate.sh?
https://github.com/apache/beam/blob/1a858c652347251e074a2ea02a7905775674520e/release/src/main/scripts/build_release_candidate.sh#L52

Reply:
I think so. If we are not ready to release 3.8, we can choose to not upload them to pypi.

Reply:
It will be the most consistent to introduce `3.8` everywhere at the same time to avoid confusion during release and to avoid filtering.

What do you think about removing `cp38`  from https://github.com/apache/beam/blob/c7b045010e9dff9fa0f5f603716c3a98de3e828b/.github/workflows/build_wheels.yml#L127 ?

Reply:
Please keep Py3.8, we should assume Py3.8 to be a supported version and we will release artifacts for Py38 with next release.